mlflow/utils/search_utils.py

```diff
@@ -1138,6 +1146,39 @@ def model_version_matches(mv):
 
         return [mv for mv in model_versions if model_version_matches(mv)]
 
+    @classmethod
+    def parse_order_by_for_search_model_versions(cls, order_by):
+        token_value, is_ascending = cls._parse_order_by_string(order_by)
+        identifier = SearchExperimentsUtils._get_identifier(
+            token_value.strip(), cls.VALID_ORDER_BY_ATTRIBUTE_KEYS
+        )
+        return identifier["type"], identifier["key"], is_ascending
+
+    @classmethod
+    def _get_sort_key(cls, order_by_list):
+        order_by = []
+        parsed_order_by = map(cls.parse_order_by_for_search_model_versions, order_by_list or [])
+        for type_, key, ascending in parsed_order_by:
+            if type_ == "attribute":
+                # Need to add this mapping because version is a keyword in sql
+                if key == "version_number":
+                    key = "version"
+                order_by.append((key, ascending))
+            else:
+                raise MlflowException.invalid_parameter_value(f"Invalid order_by entity: {type_}")
+
+        # Add a tie-breaker
+        if not any(key == "last_updated_timestamp" for key, _ in order_by):
+            order_by.append(("last_updated_timestamp", False))
```

Comment:
Can `last_updated_timestamp` really break ties? It's possible that two model versions are created within 1 millisecond. Can we use `name` and `version` instead?

Reply:
I'm trying to keep the same behavior as databricks backend, Weichen mentioned databricks backend orders by last_updated_timestamp DESC by default, and how does it deal with such scenario? By name ASC + version DESC?

Reply:
Let me check.

Reply:
The default ordering is different than a tiebreaker.

Reply:
I see, I should update the default to be ["last_updated_timestamp DESC", "name ASC", "version DESC"]